README.md

```diff
@@ -1,53 +1,112 @@
+
+> **Note:** Large Linux distros focused on usability, such as CentOS and Ubuntu, ship with many of these packages by default.
+
+Package Name | Minimum Version
+------------ |:--------------:
+**Build Dependencies** | -
+[gcc](https://gcc.gnu.org/index.html) | 6.3.1
+[cmake](https://cmake.org) | 3
+[scons](http://www.scons.org) |
+[doxygen](https://www.stack.nl/~dimitri/doxygen) |
+**OpenGL Dependencies** | -
+[libX11-devel](https://www.x.org) |
+[libXi-devel](https://www.x.org) |
+[libXmu-devel](https://www.x.org) |
+[mesa-libGL-devel](https://www.mesa3d.org) |
+[mesa-libGLU-devel](https://www.mesa3d.org) |
+**Image Format Dependencies** | -
+[nasm](https://www.nasm.us) |
+[zlib-devel](https://www.zlib.net) |
+
+
+#### Documentation Build Requirements ####
+
+> **Note:** Building the documentation is optional.
+
+Package Name | Minimum Version
+------------ |:--------------:
+[python2-pip](https://pypi.org/project/pip) |
+[sphinx](http://www.sphinx-doc.org/) | 1.4
+[inkscape](http://inkscape.org) |
+
+pip Module |
+---------- |
+sphinx_rtd_theme |
+recommonmark |
+
+
+### Build Process ###
+
+Gaffer also depends on a number of 3rd-party libraries and python modules, many of which are not entirely straightforward to build. We therefore recommend using the latest pre-built dependencies from the [Gaffer dependencies project](https://github.com/GafferHQ/dependencies/releases). These are used in our automated test builds and so are guaranteed to be up-to-date with Gaffer's requirements.
+
+Once you've downloaded the dependencies, unpack them and move/rename them to the directory in which you want to make your Gaffer build. We'll refer to this location as `<BUILD_DIR>` in the instructions below. Before continuing, make sure the dependencies are unpacked as `<BUILD_DIR>/include`, `<BUILD_DIR>/lib`, etc.
+
+Next, clone the Gaffer source and `cd` into its directory:
+
+```bash
 git clone https://github.com/GafferHQ/gaffer.git
 cd gaffer
 ```
 
-You can then build Gaffer itself as follows :
+You can then build Gaffer with `scons`:
 
-```
+```bash
 scons BUILD_DIR=<BUILD_DIR> build
 ```
 
-If you encounter any problems, please get in touch via [the developer mailing list](https://groups.google.com/forum/#!forum/gaffer-dev) and we'll do our best to help get you up and running.
 
-Contributing
-============
+## Questions and Troubleshooting ##
+
+If you have any questions about using Gaffer, or encounter problems setting it up, feel free to ask on the [Gaffer community group](https://groups.google.com/forum/#!forum/gaffer-dev). Our users and contributors are happy to help.
+
+
+## Requesting Features ##
+
+If there is a feature you would like to see in Gaffer, request it on the [Gaffer community group](https://groups.google.com/forum/#!forum/gaffer-dev). Do not create an Issue for it on GitHub.
+
+
+## Contributions and Bugs Reports ##
+
+Please see the project's [Contributing](https://github.com/GafferHQ/gaffer/blob/master/CONTRIBUTING.md) guidelines.
```

Comment:
Is this file in another PR? I didn't see it here...

Reply:
GitHub now recommends using their in-site upload feature for project description files (not entirely sure if there is a legitimate reason for this), so they've been deferred to [this ticket](http://shotgun/detail/Ticket/12540).